tests/test_server.py

```diff
@@ -52,140 +71,152 @@ def test_add_device(client, name='steelseries-rival310'):
     data = {
         'shortname': name
     }
-    response = client.post('/devices/add',
-                           data=json.dumps(data),
-                           content_type='application/json')
+    response = client.post('/devices/add', json=data)
     assert response.status_code == 201
-    data = json.loads(response.data)
+    data = response.json()
     assert 'id' in data
     assert 'shortname' in data
     assert 'name' in data
```

Comment:
you should also ensure that the created device has the correct properties, at the evdev level

Reply:
I did same as [here](https://gitlab.freedesktop.org/libevdev/python-libevdev/blob/master/examples/evtest.py#L20) and I was able to get the supported events. How can I verify if this is correct?